skyvern/services/webhook_service.py

```
@@ -0,0 +1,283 @@
+        error=error,
+    )
+
+
+async def _build_webhook_payload(organization_id: str, run_id: str) -> _WebhookPayload:
+    run = await app.DATABASE.get_run(run_id, organization_id=organization_id)
+    if not run:
+        # Attempt to resolve task v2 runs that may not yet be in the runs table.
+        task_v2 = await app.DATABASE.get_task_v2(run_id, organization_id=organization_id)
+        if task_v2:
+            return await _build_task_v2_payload(task_v2)
+        raise WebhookReplayError(f"Run {run_id} was not found.")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)